ci-operator/jobs/openstack-k8s-operators/keystone-operator/openstack-k8s-operators-keystone-operator-master-presubmits.yaml

```diff
@@ -297,6 +297,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )keystone-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
This is OK.
But we should change the original `build-deploy` job to `always_run: false` so we don ´t have duplicated jobs running on every PR

Reply:
Where should that be changed? Manually in `ci-operator/jobs/openstack-k8s-operators/keystone-operator/openstack-k8s-operators-keystone-operator-master-presubmits.yaml`?

Reply:
Yes, here[1]. So we don't have both jobs running and consuming resources.


[1] https://github.com/openshift/release/blob/ed703d4edae721c4f2ff9611da7b5173e94d272f/ci-operator/jobs/openstack-k8s-operators/keystone-operator/openstack-k8s-operators-keystone-operator-master-presubmits.yaml#L239

Reply:
ack, done